receiver/sqlserverreceiver/scraper.go

```diff
@@ -300,8 +302,24 @@ func (s *sqlServerScraperHelper) recordDatabasePerfCounterMetrics(ctx context.Co
 			} else {
 				s.mb.RecordSqlserverReplicaDataRateDataPoint(now, val.(float64), metadata.AttributeReplicaDirectionTransmit)
 			}
+		case diskReadIOSec:
+			val, err := retrieveFloat(row, valueKey)
```

Comment:
A dumb question: where can I find the expected data types for these columns? I tried running the query locally, and the values are showing as 0. It seems like there is no official documentation that describes these data types.

Reply:
Good question, the view being queried for these is [sys.dm_os_performance_counters](https://learn.microsoft.com/en-us/sql/relational-databases/system-dynamic-management-views/sys-dm-os-performance-counters-transact-sql?view=sql-server-ver16), which states the `cntr_value` is the data type `bigint`.